presto-docs/src/main/sphinx/release/release-329.rst

```diff
@@ -0,0 +1,50 @@
+=========================
+Release 329 (XXX 2020)
+=========================
+
+General Changes
+---------------
+
+* Fix incorrect result of ``last_day_of_month`` function for first day of month. (:issue:`2452`)
+* Handle common disk failures during the spill. When on of disks got broken, it Presto will use another
+  one without failing the query. (:issue:`2444`)
+* Reduce number of metadata queries when querying ``information_schema``. (:issue:`2488`)
+* Limit broadcasted table size to 100MB by default when using the ``AUTOMATIC`` join reordering strategy. (:issue:`2527`)
+
+Security Changes
+----------------
+
+* Add ``ldap.bind-dn`` and ``ldap.bind-password`` LDAP properties to allow LDAP authentication
+  access LDAP server using service account. (:issue:`1917`)
+
+CLI Changes
+-----------
+
+* Allow inspecting network requests. (:issue:`2524`)
```

Comment:
How?

Reply:
By enabling DEBUG on `io.prestosql.cli.QueryRunner` via CLI's `--log-levels-file` switch

(`presto-cli --log-levels-file <(echo "io.prestosql.cli.QueryRunner=DEBUG")`)

Reply:
maybe add "... by enabling DEBUG logging for `io.prestosql.cli.QueryRunner`" ?

Reply:
good idea

Reply:
This is not network requests, simply the URL. I think we should add first class support for printing network requests. I'm not sure this is even worth documenting.

Reply:
don't have strong opinion. I'm fine either way

Reply:
Ok, I'll remove it then